pkg/cmd/provisioning/nutanix/nutanix.go

```diff
@@ -0,0 +1,23 @@
+package nutanix
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	TargetDir      string
+	CredRequestDir string
+}
+
+// NewNutanixCmd implements the "nutanix" subcommand for the credentials provisioning
+func NewNutanixCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "nutanix",
+		Short: "Manage credentials objects for Nutanix",
+		Long:  "Creating/deleting cloud credentials objects for Nutanix",
```

Comment:
Doesn't really implement delete...
```suggestion
		Long:  "Creating cloud credentials objects for Nutanix",
```